docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc

```diff
@@ -0,0 +1,460 @@
+[role="xpack"]
+[[tutorial-migrate-data-stream-from-ilm-to-dsl]]
+=== Tutorial: Migrate ILM managed data stream to Data stream lifecycle 
+
+preview:[]
```

Comment:
Missed this before, I think it will look better if we use `preview::[]`. This would give a nice banner instead of simply saying `[preview]`. You can see it being used [here](https://elasticsearch_101117.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-time-series-aggregation.html).

Reply:
++ good catch. I'll update it everywhere as I'm in the area now. Thanks Mary